### PR TITLE
HHT-408: JsonLD serialization support

### PR DESCRIPTION
**Pull request by ktatarnikovhiro**
- JsonLD serialization support
- refined graph structure supporting Literals and IRI.
- cluster info to kg transformer

### CODE REVIEW

File: app/serialize/jsonld_serializer.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        self,
+        out_results: List[Dict[str, Any]],
+        graph: Graph,
+        queue: List[IRI],
+        all_node_ids: Set[IRI],
+        aggregate_ids: Set[IRI],
+        default_context: Dict[str, Any],
+    ) -> None:
+        while len(queue) > 0:
+            node_id, queue = queue[0], queue[1:]
```
Comment (vladimir-gladkii) [inline app/serialize/jsonld_serializer.py:60]: It looks like a usual list walk. Using "for/in" would be easier to read and a bit more efficient.

File: app/k8s_transform/transformer_base.py [hunk 2]
```diff
@@ -1,77 +1,110 @@
...
+                    self.escape(f"{self.normalize(label)}:{self.normalize(value)}"),
+                    Literal.TYPE_STRING,
+                )
             )
         self.sink.add_property_collection(name, property, subjects)
 
     def escape(self, token: str) -> str:
         token = token.replace('"', '\\"')
-        token = f'"{token}"'
+        token = f"{token}"
```
Comment (vladimir-gladkii) [inline app/k8s_transform/transformer_base.py:91]: Looks redundant
token = f"{token}"

**Review by vladimir-gladkii — DISMISSED** [review]
There are two points. Look in the comments to the code. But they are not critical.

File: app/serialize/jsonld_serializer.py (hunk 1 above)
Comment (ktatarnikovhiro) [reply app/serialize/jsonld_serializer.py]: fixed

File: app/k8s_transform/transformer_base.py (hunk 2 above)
Comment (ktatarnikovhiro) [reply app/k8s_transform/transformer_base.py]: fixed

**Review by vladimir-gladkii — APPROVED** [review]

**Pull request merged**